### PR TITLE
README: clarify documentation update after a new release

### PR DESCRIPTION
**Pull request by jouvin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,9 +88,14 @@ drop and drag them into place or just select them with the chooser.
...
-For the documentation, log into ReadTheDocs.org and add the tag for the
-release.  Once added it will build automatically.  You can then select
-the latest release as the default version to show.
+For the documentation, log into ReadTheDocs.org with an account who is
+listed as a maintainer for the `quattor-pan` project and add the tag for the
+release to the list of versions to build (this is done in the `Versions` tab
+of the `Settings` page access through the `Admin` button of the project home page).
+Once added it will build automatically.  The default version displayed on
+ReadTheDocs is normally `stable` which will automatically mapped to the new version
+once it is built (if it is the highest tag). You can also explicitely select the version
```
Comment (stdweird) [inline README.md]: `explicitely` -> `explicitly`

File: README.md [hunk 2]
```diff
@@ -88,9 +89,16 @@ drop and drag them into place or just select them with the chooser.
 Provide a title and description for the release and then click on the
 "Publish" button.
 
-For the documentation, log into ReadTheDocs.org and add the tag for the
-release.  Once added it will build automatically.  You can then select
-the latest release as the default version to show.
+After producing the release, the documentation on ReadTheDocs (RTD) must be updated,
+as RTD currently does not build automatically new versions.
```
Comment (jrha) [inline README.md]: does not **automatically build** new versions

File: README.md [hunk 3]
```diff
@@ -24,10 +24,11 @@ on a CentOS (or compatible) machine with the `rpmbuild` command installed.
 Ensure also that a recent, certified version of the JVM (1.6+) and maven
 (3.0.3+) are installed.
 
-Before doing anything else, ensure that the release notes and the change
-log for the release are up to date and complete.  The source is in the
+Before doing anything else, **ensure that the release notes and the change
+log for the release are up to date and complete**.  The source is in the
```
Comment (jrha) [inline README.md]: What is meant by "complete" here?

File: README.md [hunk 4]
```diff
@@ -88,9 +89,16 @@ drop and drag them into place or just select them with the chooser.
 Provide a title and description for the release and then click on the
 "Publish" button.
 
-For the documentation, log into ReadTheDocs.org and add the tag for the
-release.  Once added it will build automatically.  You can then select
-the latest release as the default version to show.
+After producing the release, the documentation on ReadTheDocs (RTD) must be updated,
+as RTD currently does not build automatically new versions.
+You must log into ReadTheDocs.org with an account who is
```
Comment (jrha) [inline README.md]: with an account ~who is~

File: README.md [hunk 5]
```diff
@@ -88,9 +89,16 @@ drop and drag them into place or just select them with the chooser.
...
+listed as a maintainer for the `quattor-pan` project and add the tag for the
+release to the list of versions to build (this is done in the `Versions` tab
+of the `Settings` page access through the `Admin` button of the project home page).
+Once added it will build automatically.  The default version displayed on
+ReadTheDocs is normally `stable` which will automatically mapped to the new version
+once it is built (if it is the highest tag). You can also explicitly select the version
+to show on lower-left corner.
```
Comment (jrha) [inline README.md]: This is a per-user-session setting though…

Comment (jouvin) [reply README.md]: No this is part of the project settings... A user can change this but the default is really ` stable`...

Comment (jrha) [reply README.md]: Oh I see, you mean that the project admins can change which value appears there by default.

**Pull request merged**